fbw-a32nx/src/systems/instruments/src/EWD/instrument.tsx

```diff
@@ -117,6 +117,12 @@ class A32NX_EWD extends BaseInstrument {
         this.simVarPublisher.subscribe('slatsPositionRaw');
 
         FSComponent.render(<EwdComponent bus={this.bus} instrument={this} />, document.getElementById('EWD_CONTENT'));
+
+        // Remove "instrument didn't load" text
+        document.getElementById('EWD_CONTENT').querySelector(':scope > h1').remove();
+
+        const sub = this.bus.getSubscriber<ClockEvents>();
+        sub.on('realTime').handle((deltaTime) => this.pseudoFwc.onUpdate(deltaTime));
```

Comment:
Just curious. Why is this line re-added, while it had been removed to fix some delta time issue in this PR https://github.com/flybywiresim/a32nx/pull/7801

Reply:
Indeed, that was a mistake, and even made it worse as now the FWC update method is called twice. Fixed now https://github.com/flybywiresim/a32nx/pull/7811